backend-app/utils/authorization/auth_utils.js

```diff
@@ -0,0 +1,55 @@
+const {
+    ACCESS_TOKEN_SECRET,
+    ACCESS_TOKEN_EXPIRY_TIME,
+    REFRESH_TOKEN_SECRET,
+    REFRESH_TOKEN_EXPIRY_TIME,
+    ACCESS_TOKEN_COOKIE_EXPIRY_TIME,
```

Comment:
I don't understand why would you add this isn't the ACCESS_TOKEN_EXPIRY_TIME and ACCESS_TOKEN_COOKIE_EXPIRY_TIME the same ?

Reply:
and also don't forget to add these vars to .env.local
@muttaqin1

Reply:
> I don't understand why would you add this isn't the ACCESS_TOKEN_EXPIRY_TIME and ACCESS_TOKEN_COOKIE_EXPIRY_TIME the same ?

Same ,but i have used "1d" in  ACCESS_TOKEN_EXPIRY_TIME.
```txt
ACCESS_TOKEN_EXPIRY_TIME=1d
```
 and the Cookie maxAge expects number. I was in hurry thats why i added new new env variables (ACCESS_TOKEN_COOKIE_EXPIRY_TIME) instead if changing the ACCESS_TOKEN_EXPIRY_TIME to number😅.

```txt
ACCESS_TOKEN_EXPIRY_TIME=100000000
```
Sorry about that.